docs/developing/sandbox.md

```diff
@@ -23,6 +23,30 @@ The Octez client sandboxed and mockup modes run a local version of the Tezos net
 - [Sandboxed mode](https://tezos.gitlab.io/user/sandbox.html) runs a local network with one or more nodes.
 - [Mockup mode](https://tezos.gitlab.io/user/mockup.html) runs a light version of the network without nodes.
 
+## Tezbox
+
+[Tezbox](https://github.com/tez-capital/tezbox) is also a simulated Tezos environment that runs in a container.
```

Comment:
Beware, the "is also" is misplaced, now that you swapped the sections for Tezbox & Flextesa.

Reply:
Thanks for catching that. I added a fix to this typos PR: https://github.com/trilitech/tezos-developer-docs/pull/457

Reply:
ok, mind putting the "is also" on the second section (Flextesa) :-)